torchvision/models/vision_transformer.py

```diff
@@ -328,6 +346,20 @@ class ViT_B_16_Weights(WeightsEnum):
             "acc@5": 95.318,
         },
     )
+    IMAGENET1K_SWAG_V1 = Weights(
+        url="https://download.pytorch.org/models/vit_b_16_swag-9ac1b537.pth",
+        transforms=partial(
+            ImageClassification, resize_size=384, interpolation=InterpolationMode.BICUBIC, crop_size=384
+        ),
+        meta={
+            **_COMMON_SWAG_META,
+            "num_params": 86859496,
```

Comment:
I assume this is different from the one at `IMAGENET1K_V1` because of the size of the input. Our tests currently make the assumption that all weights have the same exactly number of parameters, this is true for 99% of the models (actually this one is the first that has this issue). So we will need to update the test logic to account for deliberate differences like this. What do you propose?

Reply:
I think we can do as you said in the chat:

> The specific line on the test is written this way to avoid slowdowns related to downloading and loading the model. I think it's a good trick for 99% of the models but when it fails we should check explicitly if the number of parameters match the meta-data. Aka do the same as line: https://github.com/pytorch/vision/blob/main/test/test_extended_models.py#L115

So if the fast check fails, a more thorough check is done prior marking it as incorrect. What you think?

will add a commit for this